tsdb/wal/watcher.go

```diff
@@ -507,6 +512,23 @@ func (w *Watcher) readSegment(r *LiveReader, segmentNum int, tail bool) error {
 				send = send[:0]
 			}
 
+		case record.Exemplars:
+			// Skip if experimental "exemplars over remote write" is not enabled.
+			if !w.sendExemplars {
+				break
+			}
+			// If we're not tailing a segment we can ignore any exemplars records we see.
+			// This speeds up replay of the WAL by > 10x.
```

Comment:
```suggestion
			// This speeds up replay of the WAL.
```
I don't think it's 10x is for exemplars so let's stop fake news 🙈